WalletWasabi.Packager/MacSignTools.cs

```diff
@@ -0,0 +1,394 @@
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.IO.Compression;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Threading;
+
+namespace WalletWasabi.Packager
+{
+	public static class MacSignTools
+	{
+		public static void Sign()
+		{
+			if (!RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
+			{
+				throw new NotSupportedException("This signing methon only valid on macOS!");
```

Comment:
```suggestion
				throw new NotSupportedException("This signing method is only valid on macOS!");
```

Reply:
I saw below that there is a `IsMacSignMode` method. Does it make sense to use it here?

Reply:
No. At some point I will have to add cmd line argument which will determine what to do with packager. But for now I just check that we are on mac or not. 
This platform check is a sanity check of this function.